doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -22,6 +21,7 @@ The relative width and height of columns and rows in `~.Figure.subplots` and
 *width_ratios* keyword arguments to the methods:
 
 .. plot::
+    :alt: A figure with 3 subplots in 3 rows and 1 column. The hight of the subplot in the first row is 3 times more than the subplots in the 2nd and 3rd row
```

Comment:
hight -> height.

Also, it's not 3 times _more_; it's 3 times exactly. Please spell out second and third as well.

Reply:
I would add that they are "empty", but if this is imprecise language maybe we don't need it.